### DISCUSSION TITLE
Menu gets cut short

### DISCUSSION THREAD

**Discussion by NYIGCommunications** [code4recovery/12-step-meeting-list#847] [General]
Hello y'all, when we're looking at the new list, if there is a selection that produces a short list, then the drop down menu for region or type, if selected, does not scroll all the way down. I hope I explained that well. Secondarily, the regional list is not organized by Borough anymore, rather all by neighborhood in alphabetical (which is not the preference). We haven't gone live yet with the new list, and it's on hold until these things can be sorted out. the usual is www.nyintergroup.org if you'd like to see the region menu as it is.

**Comment by kiyote33** [comment]
Hello @NYIGCommunications,
Excuse me, I'm a little confused with your question! When you say "new list", are you referring to the **TSML UI** User Interface Display which is set on our Settings page? If you are, I will need to transfer this issue to the "TSML UI" issue page so it get's visibility to our team members who ### handle issues that come up with the TSML UI interface. This is the Issue listings for things that come up with our "12 Step Meeting List Feedback Enhancement" plugin.

**Comment by NYIGCommunications** [comment]
Thank you for sending it along!

Regards,
Ernest Garcia (not for publication)
Communications Coordinator
Inter-Group Association of A.A. of New York, Inc.
307 Seventh Avenue, Suite 201
New York, NY 10001
***@***.******@***.***>
212-647-1647

Support NY Inter-Group
https://www.nyintergroup.org/product-category/contributions/
Venmo: @nyintergroup
Paypal: https://www.paypal.me/nyintergroup

**Comment by joshreisner** [comment]
Thanks @NYIGCommunications - Sounds like there are two separate issues:

**on a short page, the drop down menu does not scroll all the way down**
this sounds like it needs to be resolved with CSS on your development site. [here's an example of a short page](https://aasepia.org/meetings/?region=bucks-county-croydon), try clicking the big region dropdown, you'll see that it can scroll all the way

**the regional list is not organized by Borough anymore**
NYIG has three levels of regional hierarchy, but the plugin only creates two levels in its JSON, so it needs to support more levels. I've created a ticket for this work: https://github.com/code4recovery/12-step-meeting-list/issues/846 - this will need to be done before you can see the full region dropdown in the right order

**Comment by NYIGCommunications** [comment]
Josh,

Thank you so much!! Have a good weekend.

Regards,
Ernest Garcia (not for publication)
Communications Coordinator
Inter-Group Association of A.A. of New York, Inc.
307 Seventh Avenue, Suite 201
New York, NY 10001
***@***.******@***.***>
212-647-1647

Support NY Inter-Group
https://www.nyintergroup.org/product-category/contributions/
Venmo: @nyintergroup
Paypal: https://www.paypal.me/nyintergroup
